includes/admin/dashboard-widgets.php

```diff
@@ -149,48 +149,60 @@ function edd_load_dashboard_sales_widget( ) {
-				<tbody>
-					<?php
-					foreach ( $payments as $payment ) { ?>
-						<tr>
-							<td class="edd_order_label">
-								<a href="<?php echo add_query_arg( 'id', $payment->ID, admin_url( 'edit.php?post_type=download&page=edd-payment-history&view=view-order-details' ) ); ?>">
-									<?php echo get_the_title( $payment->ID ) ?>
-									&mdash; <?php echo $payment->email ?>
-								</a>
-								<?php if ( ! empty( $payment->user_id ) && ( $payment->user_id > 0 ) ) {
-									$user = get_user_by( 'id', $payment->user_id );
-									if ( $user ) {
-										echo "(" . $user->data->user_login . ")";
-									}
-								} ?>
-							</td>
-							<td class="edd_order_price">
-								<a href="<?php echo add_query_arg( 'id', $payment->ID, admin_url( 'edit.php?post_type=download&page=edd-payment-history&view=view-order-details' ) ); ?>">
-									<span class="edd_price_label"><?php echo edd_currency_filter( edd_format_amount( $payment->total ), edd_get_payment_currency_code( $payment->ID ) ); ?></span>
-								</a>
-							</td>
-						</tr>
+		if ( $orders ) { ?>
+		<div class="table recent_orders">
+			<h3><?php esc_html_e( 'Recent Orders', 'easy-digital-downloads' ); ?></h3>
+			<ul>
+			<?php
+			foreach ( $orders as $order ) {
+				$link = edd_get_admin_url(
+					array(
+						'page' => 'edd-payment-history',
+						'view' => 'view-order-details',
+						'id'   => $order->ID,
+					),
+					admin_url( 'edit.php' )
+				);
+				?>
+				<li class="edd_order_label">
+					<a href="<?php echo esc_url( $link ); ?>">
 						<?php
-					} // End foreach ?>
-				</tbody>
-			</table>
+						$customer      = edd_get_customer( $order->customer_id );
+						$customer_name = $customer->name ? $customer->name : __( 'No Name', 'easy-digital-downloads' );
+						$item_counts   = edd_count_order_items( array( 'order_id' => $order->ID ) );
+						echo wp_kses_post(
+							sprintf(
+								/* translators: 1. customer name; 2. number of items purchased; 3. order total */
+								_n(
+									'%1$s purchased %2$s item for <strong>%3$s</strong>',
+									'%1$s purchased %2$s items for <strong>%3$s</strong>',
+									$item_counts,
+									'easy-digital-downloads'
+								),
+								$customer_name,
+								$item_counts,
+								edd_currency_filter( edd_format_amount( edd_get_order_total( $order->ID ) ) )
+							)
+						);
+						?>
+					</a>
+					<br /><?php echo esc_html( edd_date_i18n( $order->date_completed ) ); ?>
+				</li>
+				<?php } // End foreach ?>
+		</ul>
+			<?php
+			$all_orders_link = edd_get_admin_url(
+				array(
+					'page' => 'edd-payment-history',
+				),
+			);
+			?>
+		<a href="<?php echo esc_url( $all_orders_link ); ?>" class="button-secondary"><?php esc_html_e( 'View All Orders', 'easy-digital-downloads' ); ?></a>
 		</div>
 		<?php } // End if ?>
-		<?php do_action( 'edd_sales_summary_widget_after_purchases', $payments ); ?>
+		<?php do_action( 'edd_sales_summary_widget_after_purchases', $orders ); ?>
```

Comment:
I haven't actually checked, but are there any backwards compatibility concerns with changing this hook parameter? @spencerfinnell

Reply:
I wondered the same thing, but _assumed_ we had similar changes in `release/3.0` already. Looking now, I don't think so! And it's an easy switch back.

Pushed an update.